### PR TITLE
Heatmap sort

### PR DESCRIPTION
**Pull request by philippjfr**
Various fixes to ensure HeatMaps aren't sorted by default. May affect some tests.

### CODE REVIEW

**Comment by philippjfr** [comment]
Ready to review/merge.

File: doc/reference_data [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit c6f0638fb47810929048346f5359556c83ba448d
+Subproject commit 94a283186c7421cd25bfbb56774d986fce2c4c3e
```
Comment (jlstevens) [inline doc/reference_data]: Just curious...how did the reference data change?

Comment (philippjfr) [inline doc/reference_data]: Not quite clear on why it changed as the visual output hasn't and the Elements example didn't require any change. But we'll delete this Tutorial very soon anyway and regenerate the reference_data.

**Comment by jlstevens** [comment]
All looks good. Merging.

**Pull request merged**